### ISSUE TITLE
Update identifiers specified in CIP5 with prefixed bech32 encoding

### ISSUE THREAD

**Issue by rhyslbw** [input-output-hk/cardano-node#1835]
Other tools and services are in the process of implementing [CIP5](https://github.com/cardano-foundation/CIPs/tree/master/CIP5), despite it's draft status, so to ensure we're presenting a consistent value, it would be good timing to implement here:

## Applicable Prefixes 
- [ ] `pool`
- [ ] `pool_vk`
- [ ] `vrf_vk`

**Comment by rhyslbw** [comment]
@erikd Are these not already implemented in the API library?
- [pool](https://github.com/input-output-hk/cardano-node/blob/master/cardano-api/src/Cardano/Api/Typed.hs#L4390-L4392)
- [pool_vk](https://github.com/input-output-hk/cardano-node/blob/master/cardano-api/src/Cardano/Api/Typed.hs#L4371-L4373)
- [vrf_vk](https://github.com/input-output-hk/cardano-node/blob/master/cardano-api/src/Cardano/Api/Typed.hs#L4566-L4568)

**Comment by intricate** [comment]
@rhyslbw: Yes, the API already has support for the Bech32 prefixes specified in CIP5 (including the ones you've listed).

**Comment by dcoutts** [comment]
We recently audited the `cardano-api` for compliance with CIP5.

**Comment by rhyslbw** [comment]
Thanks @intricate and @dcoutts. Please transfer this issue back to `cardano-db-sync` as I don't have maintainer capabilities to do it myself
